app/data/jsonSchemaForm/fundingAgreementSchema.ts

```diff
@@ -48,6 +48,8 @@ export const fundingAgreementSchema = {
 
 export const fundingAgreementUiSchema = {
   "ui:order": [
+    "contractStartDate",
+    "projectAssetsLifeEndDate",
```

Comment:
these lines are duplicated instead of reordered which is causing the unit and e2e tests to fail

Reply:
Good catch, thanks!